pkg/services/sqlstore/migrations/libraryelements.go

```diff
@@ -61,4 +61,11 @@ func addLibraryElementsMigrations(mg *migrator.Migrator) {
 
 	mg.AddMigration("alter library_element model to mediumtext", migrator.NewRawSQLMigration("").
 		Mysql("ALTER TABLE library_element MODIFY model MEDIUMTEXT NOT NULL;"))
+
+	mg.AddMigration("add library_element folder uid", migrator.NewAddColumnMigration(libraryElementsV1, &migrator.Column{
```

Comment:
We have a unique index on `org_id, folder_id, name, kind`. With this new column, we could probably want a similar index on `org_id, folder_uid, name, kind`.